src/transformers/modeling_utils.py

```diff
@@ -780,7 +775,7 @@ def get_extended_attention_mask(
         # Since we are adding it to the raw scores before the softmax, this is
         # effectively the same as removing these entirely.
         extended_attention_mask = extended_attention_mask.to(dtype=self.dtype)  # fp16 compatibility
-        extended_attention_mask = (1.0 - extended_attention_mask) * -10000.0
+        extended_attention_mask = (1.0 - extended_attention_mask) * torch.finfo(dtype).min
```

Comment:
Add a `dtype` argument, so we can use it in `LayoutLMv3Model`

Reply:
The other 2 changes (not shown above)

https://github.com/huggingface/transformers/blob/de8beb96c1d0d323d5d09a85438b47b47a52b0ac/src/transformers/modeling_utils.py#L730

https://github.com/huggingface/transformers/blob/de8beb96c1d0d323d5d09a85438b47b47a52b0ac/src/transformers/modeling_utils.py#L744-L745